papers/agrawala/article.md

```diff
@@ -4,9 +4,15 @@ abstract: |
-Machine learning algorithms can be used to predict the spectral features of the NMR spectra of an organic compound based on its chemical structure {cite:p}`https://doi.org/10.1021/acs.jcim.0c00195`. The algorithm would learn the relationship between atoms and bonds in a molecule, and how it affects the NMR spectra of that molecule. Simple neural network algorithms simplify molecules by encapsulating each atom’s features within a vector, albeit at the cost of losing significant information about interatomic interactions. However, a more effective approach to representing molecules for machine learning algorithms emerges with the utilization of graphs {cite:p}`https://doi.org/10.1021/acs.jcim.0c00195`. The graph is composed of nodes, which represent the atoms, that are connected by edges, which represent the bonds between atoms. A graph neural network is a machine learning algorithm that can be used to simulate a compound by mapping out the molecular structure of the compound on a graph. Furthermore, they can be used to predict the chemical shifts of 13C NMR spectra for carbon compounds more accurately than a classic neural network {cite:p}`https://doi.org/10.1021/acs.jcim.0c00195`. The graph neural network algorithm can help scientists while they are producing a new compound as they could use the algorithm to verify if they made the correct compound.
+Machine learning algorithms can be used to predict the spectral features of the NMR spectra of an organic compound based on its chemical structure {cite:p}`https://doi.org/10.1021/acs.jcim.0c00195`. The algorithm would learn the relationship between atoms and bonds in a molecule, and how it affects the NMR spectra of that molecule. Simple neural network algorithms simplify molecules by encapsulating each atom’s features within a vector, albeit at the cost of losing significant information about interatomic interactions. However, a more effective approach to representing molecules for machine learning algorithms emerges with the utilization of graphs {cite:p}`https://doi.org/10.1021/acs.jcim.0c00195`. The graph is composed of nodes, which represent the atoms, that are connected by edges, which represent the bonds between atoms. A graph neural network is a machine learning algorithm that can be used to simulate a compound by mapping out the molecular structure of the compound on a graph. Furthermore, they can be used to predict the chemical shifts of 13C NMR
+
++++
```

Comment:
To improve PDF formatting, I added these additional breaks. This is currently impacting the web builds, but I will fix that.